libcxx/test/libcxx/containers/views/views.span/span.cons/assert.iter_size.pass.cpp

```diff
@@ -25,13 +25,48 @@
 #include "check_assertion.h"
 
 int main(int, char**) {
-    std::array<int, 3> array{0, 1, 2};
+  std::array<int, 3> array{0, 1, 2};
 
-    auto too_large = [&] { std::span<int, 3> const s(array.data(), 4); (void)s; };
-    TEST_LIBCPP_ASSERT_FAILURE(too_large(), "size mismatch in span's constructor (iterator, len)");
+  // Providing too large value in constructor
```

Comment:
Optional: how about changing these comments slightly to something like:
```cpp
// Input range too large (exceeds the span extent)
// Input range too small (doesn't fill the span)
// Input range is non-empty but starts with a null pointer
```
?

Reply:
That's so much better than what I had! Applying.